experiment/kind-conformance-image-e2e.sh

```diff
@@ -47,25 +47,8 @@ install_kind() {
 
 # build kubernetes / node image, e2e binaries
 build() {
-    # possibly enable bazel build caching before building kubernetes
-    BAZEL_REMOTE_CACHE_ENABLED=${BAZEL_REMOTE_CACHE_ENABLED:-false}
-    if [[ "${BAZEL_REMOTE_CACHE_ENABLED}" == "true" ]]; then
-        # run the script in the kubekins image, do not fail if it fails
-        /usr/local/bin/create_bazel_cache_rcs.sh || true
-    fi
-
     # build the node image w/ kubernetes
-    kind build node-image --type=bazel --kube-root="${PWD}"
-
-    # try to make sure the kubectl we built is in PATH
```

Comment:
we may or may not need to do something like this again (probably not)
this job doesn't really depend on the local kubectl except to install the conformance pod anyhow though.

this snippet is based on the other kind jobs which actually do need to use the just-built kubectl for tests.

Reply:
implemented this, tested this part locally.